pkg/process/procutil/process_model.go

```diff
@@ -94,6 +171,14 @@ type IOCountersStat struct {
 	WriteBytes int64
 }
 
+// IOCountersRateStat holds IO metrics for a process represented as rates (/sec)
```

Comment:
nit: let's not insert this definition between `IOCountersStats` and `func (i *IOCountersStat) IsZeroValue()` which ideally should be together.

Reply:
Makes sense, it's an oversight on my part.